ebuild-writing/common-mistakes/text.xml

```diff
@@ -380,6 +380,19 @@ Make sure when you bump a version, the stable KEYWORDS are all marked as
 
 </body>
 </subsection>
+<subsection>
+<title>Unused flags in IUSE</title>
+<body>
+
+<p>
+Sometimes vestigal USE flags remain in IUSE despite having no function. Remember
+to prune these.
+</p>
+
+</body>
+</subsection>
+
+
```

Comment:
No double blank lines, please (the style guide even doesn't require a single one before `<subsection>`.